### PR TITLE
[ID-3] Created InventoryPage.java. Added login method

### PR DESCRIPTION
**Pull request by eugenekr**

### CODE REVIEW

File: src/test/java/pages/InventoryPage.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
```
Comment (KarynaMashkanava) [inline src/test/java/pages/InventoryPage.java]: delete import

File: src/test/java/pages/InventoryPage.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+import org.openqa.selenium.support.FindBy;
```
Comment (KarynaMashkanava) [inline src/test/java/pages/InventoryPage.java]: delete import

File: src/test/java/pages/LoginPage.java [hunk 3]
```diff
@@ -4,17 +4,24 @@
...
-    public LoginPage(WebDriver driver) {
-        super(driver);
-    }
+  public LoginPage(WebDriver driver) {
+    super(driver);
+  }
+
+  public InventoryPage login(String username, String password) {
+    usernameInput.sendKeys(username);
+    passwordInput.sendKeys(username);
```
Comment (KarynaMashkanava) [inline src/test/java/pages/LoginPage.java]: (password) instead of (username)

File: src/test/java/pages/InventoryPage.java (hunk 1 above)
Comment (eugenekr) [reply src/test/java/pages/InventoryPage.java]: fixed

File: src/test/java/pages/InventoryPage.java (hunk 2 above)
Comment (eugenekr) [reply src/test/java/pages/InventoryPage.java]: fixed

**Review by KarynaMashkanava — APPROVED** [review]

**Pull request merged**